### ISSUE TITLE
Copyright: False positive 

### ISSUE THREAD

**Issue by bcollamore** [philips-software/roslyn-analyzers#266]
for blank lines before copyright inside a region

**Comment by ynsehoornenborg** [comment]
You mean like so:
```cs
#region

// Copyright 2023 Philips N.V.
#endregion
```
